nilearn/_utils/data_gen.py

```diff
@@ -811,179 +860,563 @@ def create_fake_bids_dataset(base_dir='',
 
     n_sub : :obj:`int`, optional
         Number of subjects to be simulated in the dataset.
-        Default=10.
 
     n_ses : :obj:`int`, optional
         Number of sessions to be simulated in the dataset.
         Ignored if no_session=True.
-        Default=2.
 
     tasks : :obj:`list` of :obj:`str`, optional
         List of tasks to be simulated in the dataset.
-        Default=["localizer", "main"].
```

Comment:
Need to get back to this discussion about defaults.

Reply:
Maybe easier if I leave them be for now and we come back to it in another PR.

Reply:
I think I added back all the ones I had removed.